### PR TITLE
Mark STPPinManagementService deprecated & suggest alternative

### PR DESCRIPTION
**Pull request by vbc-stripe**
## Motivation
https://jira.corp.stripe.com/browse/RUN_ISSUING_TED-2233

### CODE REVIEW

File: Stripe/StripeiOS/Source/STPPinManagementService.swift [hunk 1]
```diff
@@ -13,6 +13,7 @@ import Foundation
 import UIKit
 
 /// STPAPIClient extensions to manage PIN on Stripe Issuing cards
+@available(*, deprecated, message: "Please use Issuing Elements instead: https://stripe.com/docs/issuing/elements")
```
Comment (wooj-stripe) [inline Stripe/StripeiOS/Source/STPPinManagementService.swift:16]: 1. The link leads to a page which seems to focus on a web integration?
2. I would expect that the deprecation message to tell users what other class/integration point to use?

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,6 +9,7 @@
 ### PaymentsUI
 * [Added] Adds support for [card brand choice](https://stripe.com/docs/card-brand-choice) to STPPaymentCardTextField and STPCardFormView. To set a default preferred network for these UI elements, use the new `preferredNetworks` parameter.
 
+* [Deprecated] Mark STPPinManagementService deprecated & suggest alternative.
```
Comment (wooj-stripe) [inline CHANGELOG.md:12]: 1. Will this require a major version change, or a minor version is acceptable?
2. Do we need to update the MIGRATING.md file? 
3. Seems like we've avoided using [Deprecated], and use [Changed] instead. ([example](https://github.com/stripe/stripe-ios/blob/master/CHANGELOG.md?plain=1#L372))

File: Stripe/StripeiOS/Source/STPPinManagementService.swift (hunk 1 above)
Comment (vbc-stripe) [reply Stripe/StripeiOS/Source/STPPinManagementService.swift:16]: Yeah that's intended, we don't have an native Issuing Elements yet so this is the only option at the moment. 

Please https://jira.corp.stripe.com/browse/RUN_ISSUING_TED-2233 for context

File: CHANGELOG.md (hunk 2 above)
Comment (vbc-stripe) [reply CHANGELOG.md]: done

**Review by wooj-stripe — APPROVED** [review]

**Pull request merged**